### PR TITLE
Add JC filter plugin example

### PR DESCRIPTION
**Pull request by kellyjonbrazil**
##### SUMMARY
Added an example using the JC filter plugin.

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
The JC filter plugin has been available in the community.general collection for some time now, so I thought it would be good to add an example to the existing documentation site.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
community.general.jc

##### ADDITIONAL INFORMATION
This is a documentation addition. Let me know if there is a better place for this example. Thanks!

### CODE REVIEW

**Comment by kellyjonbrazil** [comment]
+label: ready_for_review

File: docs/docsite/rst/network/user_guide/cli_parsing.rst [hunk 1]
```diff
@@ -56,6 +56,8 @@ The ``cli_parse`` module includes the following cli_parsing plugins:
   A library for semi-structured text parsing using templates, with added capabilities to simplify the process
 ``pyats``
   Uses the parsers included with the Cisco Test Automation & Validation Solution
+``jc``
+  A python module which converts the output of dozens of popular linux/unix/macOS/Windows commands and file-types to python dictionaries or lists of dictionaries. Note: this filter plugin can be found in the ``community.general`` collection.
```
Comment (Akasurde) [inline docs/docsite/rst/network/user_guide/cli_parsing.rst]:
```suggestion
  A python module that converts the output of dozens of popular Linux/UNIX/macOS/Windows commands and file types to python dictionaries or lists of dictionaries. Note: this filter plugin can be found in the ``community.general`` collection.
```

File: docs/docsite/rst/network/user_guide/cli_parsing.rst [hunk 2]
```diff
@@ -583,6 +585,29 @@ The task sets the follow fact as the ``interfaces`` fact for the host:
      state: up
 
 
+Parsing with JC
+-----------------
+
+JC is a Python library that converts the output of dozens of common linux/unix/macOS/Windows command-line tools and file-types to python dictionaries or lists of dictionaries for easier parsing. JC is available as a filter plugin in the ``community.general`` collection.
```
Comment (Akasurde) [inline docs/docsite/rst/network/user_guide/cli_parsing.rst]:
```suggestion
JC is a Python library that converts the output of dozens of common Linux/UNIX/macOS/Windows command-line tools and file-types to python dictionaries or lists of dictionaries for easier parsing. JC is available as a filter plugin in the ``community.general`` collection.
```

**Comment by Akasurde** [comment]
@kellyjonbrazil Thanks for submitting this PR. A few nit-picks, rest LGTM. Waiting for others to comment.

**Review by Akasurde — APPROVED** [review]
LGTM

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @kellyjonbrazil for the Ansible fix!

**Pull request merged**